bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -143,6 +143,17 @@ def test_projection(datapath):
     _test_equal(bodo_df2, py_df2, check_pandas_types=False)
 
 
+def test_filter(datapath):
+    """Very simple test for projection for sanity checking."""
+    bodo_df1 = bd.read_parquet(datapath("dataframe_library/df1.parquet"))
+    bodo_df2 = bodo_df1[bodo_df1.A < 20]
```

Comment:
Assert that `bodo_df2` isn't evaluated yet?

Reply:
Done.